### PR TITLE
Add optional dark theme

### PR DESCRIPTION
**Pull request by EIREXE**
Pretty hacky, but it works.

### CODE REVIEW

File: templates/layout.html [hunk 1]
```diff
@@ -13,11 +13,15 @@
         <title>Kerbal Stuff</title>
         {% endblock %}
         <link href="/static/bootstrap.min.css" rel="stylesheet">
+        {% block styles %}
+        {% endblock %}
+        {% if user.dark_theme == True %}
```
Comment (SirCmpwn) [inline templates/layout.html]: `{% if user.dark_theme %}` would be better here

File: KerbalStuff/blueprints/profile.py [hunk 2]
```diff
@@ -54,6 +54,7 @@ def profile(username):
         profile.backgroundMedia = request.form.get('backgroundMedia')
         bgOffsetX = request.form.get('bg-offset-x')
         bgOffsetY = request.form.get('bg-offset-y')
+        profile.dark_theme = request.form.get('dark_theme')
```
Comment (SirCmpwn) [inline KerbalStuff/blueprints/profile.py]: iirc this doesn't come back as a boolean. Are you sure this works?

**Comment by SirCmpwn** [comment]
All database changes require an alembic migration.

**Comment by SirCmpwn** [comment]
Doesn't seem to work when I tested this locally.

File: KerbalStuff/blueprints/profile.py (hunk 2 above)
Comment (EIREXE) [inline KerbalStuff/blueprints/profile.py]: I'm using this live on my own server and it works.

**Comment by EIREXE** [comment]
What do you mean? I created the dark_theme entry manually and I made it default to false in postgres.

**Comment by SirCmpwn** [comment]
http://alembic.readthedocs.org/en/rel_0_7/

**Comment by EIREXE** [comment]
This way?

**Comment by SirCmpwn** [comment]
Yep. But there's still the problem of this not actually working when I tested it.

**Comment by EIREXE** [comment]
So enabling it in your profile does not work? it's strange because it's a 1:1 copy of the code I use in my fork.

**Comment by SirCmpwn** [comment]
Yeah, it's weird. The logo moves with that weird margin you put in, but the appearance of the site is unchagned.

**Comment by EIREXE** [comment]
I just tested it and it seems to work after enabling it in the profile.

http://i.imgur.com/VRW8w5k.png

**Comment by SirCmpwn** [comment]
It's getting late. I'll test this more tomorrow.

**Comment by EIREXE** [comment]
Alright.

**Comment by SirCmpwn** [comment]
Gave this a fresh review. Still does not work.

**Comment by SirCmpwn** [comment]
Oh, I see. You added `bootstrap_dark.min.css` to `static`. The `static` directory gets overwritten. I'll put it in the right place.

**Pull request merged**